hooks/boilerplate.py.txt

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
No 2014 is right...   We regex all 2015->2014 before we compare with the boilerplate

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/hooks/boilerplate.py#L54